package.json

```diff
@@ -22,6 +22,7 @@
     "lint": "eslint . --ext .ts --fix",
     "lintNoFix": "eslint . --ext .ts",
     "test": "nyc mocha",
+    "prepare": "git submodule update --init --recursive && npm run build",
```

Comment:
Added this prepare script to trigger `xpring-js` build when pulling this in locally as a `git` branch through `npm`.

"NOTE: If a package being installed through git contains a prepare script, its dependencies and devDependencies will be installed, and the prepare script will be run, before the package is packaged and installed."

From: https://docs.npmjs.com/cli/v6/using-npm/scripts (for more information on what the prepare script does)

Reply:
Is this still a useful thing to have generally?  Or irrelevant outside the context of using xpring-js as a git branch?

Reply:
The things it does:

```
Runs BEFORE the package is packed
Runs BEFORE the package is published
Runs on local npm install without any arguments
Run AFTER prepublish, but BEFORE prepublishOnly
NOTE: If a package being installed through git contains a prepare script, its dependencies and devDependencies will be installed, and the prepare script will be run, before the package is packaged and installed.
```

I would say probably irrelevant unless one of the use cases stands out to you, mainly because you already have a `prePublishOnly` script anyways.

Reply:
Yeah, this _seems_ irrelevant to me since we're getting away from the branch dependency model.